use-cases/predictive_maintenance/3_train_tune_predict_predmaint.ipynb

```diff
@@ -4,43 +4,58 @@
    "cell_type": "markdown",
```

Comment:
<p>Do you want to mention the "Part 1" / "Part 2" parts of the notebook names to more explicitly illustrate the order of these notebooks?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3365/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>This is a good point to bring up. Personally, I don't think we should explicitly name them Part 1 and Part 2 because I want the notebooks to be able to used independently (i.e. person just wants to learn about featurization) or in sequence (which is always clearly specified in the "Background" Section of each notebook in the series). But I will make it clearer in the comments that one notebook follows the other.</p>

Reply:
<p>Added more details, let me know what you think</p>